lib/cdo/github.rb

```diff
@@ -100,4 +100,26 @@ def self.sha(branch)
     response = Octokit.branch(REPO, branch)
     response.commit.sha
   end
+
+  # Opens a browser URL with a candidate pull request merging head into base.
+  # @param base [String] The base branch of the comparison.
+  # @param head [String] The head branch of the comparison.
+  # @param title [String] The title of the candidate pull request.
+  # @raise [RuntimeError] If the environment is not development.
+  def self.open_pull_request_in_browser(base:, head:, title:)
```

Comment:
Nit: This name isn't exactly true, since we're opening the compare page to _help_ create a pull request.